docs/pages/index.mdx

```diff
@@ -64,25 +54,26 @@ Get started with Teleport Access:
   to access infrastructure resources with IdPs like Okta.
 - [Use Teleport as an identity provider](admin-guides/access-controls/idps/saml-guide.mdx) to
   authenticate to external services.
-- [Issue and govern identities for automated
-  systems](enroll-resources/machine-id/introduction.mdx) using Machine ID.
 
-### Teleport Identity
+### Teleport Machine & Workload Identity
+
+Teleport Machine & Workload Identity is a solution for non-human identity management and access control, 
+improving infrastructure resiliency by securing system and data access between machines and workloads. 
+As part  of Teleport's Infrastructure Identity Platform, Teleport Machine & Workload Identity is purpose-built 
+for infrastructure and is part of a unified access control strategy for both human and non-human identities.
+
+Get started with Teleport Machine & Workload Identity:
+
+- [Issue and govern identities for automated systems](enroll-resources/machine-id/introduction.mdx) using Teleport Machine Identity.
+- Replace long-lived API keys and environment secrets that are vulnerable to breaches, mistakes, and exfiltration with short-lived cryptographic identities using x.509 certificates or JWTs via Teleport Workload Identity.
```

Comment:
This still needs the link from the Google Doc: https://goteleport.com/docs/enroll-resources/workload-identity/introduction/